modules/vitaliy/src/main/java/com/hillel/elementary/javageeks/vitaliy/task1/Times.java

```diff
@@ -0,0 +1,13 @@
+package com.hillel.elementary.javageeks.vitaliy.task1;
+
+import java.util.Date;
+
+public class Times {
+    public static void main(String args[]) {
+        // Инициализация объекта date
+        Date date = new Date();
+
+        // Вывод текущей даты и времени с использованием toString()
```

Comment:
I think you don`t really need "\n". println output data from a new line by itself